### ISSUE TITLE
Time and random command-args in groups

### ISSUE THREAD

**Issue by loopier** [loopier/animatron-godot3#33]
I found that it's quite nice to send commands to the members of a group with time intervals, in addition to all at once -- as we would do with `/scale groupName 0.5`.

It's also nice to be able to randomize the command arguments of each member individually with one single command.

Currently I do both from SC with a `Tdef` with something like this:
```
~num = 10;
x.("/create/group", "ns", "numbers");
(
Tdef(\agroup, {
    ~num(or inf).do{ |i|
        var actor = "ns_"++i.mod(~num).asInteger;
        x.("/scale", actor, rrand(0.5,1.5));
        rrand(0.1,0.3).wait;
}).play;
)
```

I think it would be useful to add this, but I haven't thought about the syntax.

**Comment by loopier** [comment]
> It's also nice to be able to randomize the command arguments of each member individually with one single command.

This is implmented in 84b48027c398d4622ec8e1c760d593da144d05eb

**Comment by loopier** [comment]
I have been able to make proto-routines with `def`s calling themselves after waiting:
```
def /bla actor
    /rand /position/x $actor 0 1
    /wait 1
    /bla $actor
```
This seems to work fine. The only problem is that they cannot be stopped. They keep running for ever.
A possible solution would be to use waits, maybe `/wait/named <NAME> <SECONDS>` then we could use `/wait/free <NAME>` to remove them.
Looking at the current code for `/wait` I saw that it has a special treatment, my guess being because it needs to yield command calls. Any idea on how we could fit timing controls into the current implementation (or any new one)?
